BrainPortal/app/controllers/tasks_controller.rb

```diff
@@ -1502,6 +1502,7 @@ def handle_preset_actions #:nodoc:
       if (! preset_id.blank?) && preset = CbrainTask.where(:id => preset_id, :status => [ 'Preset', 'SitePreset' ]).first
         old_params = @task.params.clone
         @task.params         = preset.params
+        @task.description    = preset.description
```

Comment:
Can we get a description merge maybe? This will just replace outright whatever other description the user has already entered in the task, or whatever description was generated automatically by the task (e.g. a CIVET task puts the name of the mnc file in the description by default, and that's useful for user, but if the user loads a preset now, they lose that information)

Reply:
I'd add the preset's description AFTER the original description, and with a newline added, but only of the original description is non-blank. Shoudl be possible in one line.

Reply:
might be useful, yet even config name is overwritten during merge

Reply:
@prioux, regarding the following: 

>e.g. a CIVET task puts the name of the mnc file in the description by default, and that's useful for user, but if the user loads a preset now, they lose that information

--> Civet include this information only if the description is empty, no ?

Reply:
> might be useful, yet even config name is overwritten during merge. For greater consistency maybe concatenate the config names (versions) too?

@MontrealSergiy Which config name ?

Reply:
sorry, Natacha, that was not well thought through, forget that comment